yagna_dapp_manager/dapp_manager.py

```diff
@@ -16,6 +16,9 @@
 PathType = Union[str, os.PathLike]
 
 
+from dapp_runner import _util
```

Comment:
I think this is not used now?

Reply:
that's some glitch

Reply:
aaah, not a glitch but a remainder of my "hack" ;p, removing immediately